src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -80,7 +80,7 @@ public Command parseCommand(String userInput) throws ParseException {
             return new HelpCommand();
 
         case CountCommand.COMMAND_WORD:
-            return new CountCommand();
+            return new CountCommandParser().parse(arguments);
```

Comment:
Good usage of abstracting the command to make it fit with the rest of the logic

Reply:
Well done on making the code more maintainable and extensible